content/en/tracing/connect_logs_and_traces/ruby.md

```diff
@@ -37,6 +40,9 @@ config.lograge.custom_options = lambda do |event|
   {
     # Adds IDs as tags to log output
     :dd => {
+      :env => correlation.env,
+      :service => correlation.service,
+      :version => correlation.version,
       :trace_id => correlation.trace_id,
```

Comment:
This is incorrect; all numeric IDs should be coerced to strings so as to not lose precision. This was updated in https://github.com/DataDog/dd-trace-rb/pull/826 because users had this issue, and we added a comment to make it clear to customers why this was important.

All of these values should be suffixed by `.to_s`.